components/dash-core-components/tests/integration/markdown/test_markdown.py

```diff
@@ -93,3 +93,56 @@ def test_mkdw002_dcclink(dash_dcc):
     dash_dcc.percy_snapshot("mkdw002 - markdowns display")
 
     assert dash_dcc.get_logs() == []
+
+
+def test_mkdw003_without_mathjax(dash_dcc):
+    app = Dash(__name__, eager_loading=False, assets_folder="../../assets")
+
+    app.layout = html.Div(
+        [
+            dcc.Markdown("# No MathJax: Apple: $2, Orange: $3"),
+        ]
+    )
+
+    dash_dcc.start_server(app)
+    dash_dcc.percy_snapshot("mkdw003 - markdown without mathjax")
+    assert dash_dcc.get_logs() == []
+
+
+def test_mkdw004_inline_mathjax(dash_dcc):
+    app = Dash(__name__, eager_loading=False, assets_folder="../../assets")
+
+    app.layout = html.Div(
+        [
+            dcc.Markdown("# h1 tag with inline MathJax: $E=mc^2$", mathjax=True),
+        ]
+    )
+
+    dash_dcc.start_server(app)
+    dash_dcc.percy_snapshot("mkdw004 - markdown inline mathjax")
```

Comment:
This part is visual, so a snapshot makes sense, but I'd probably combine this and the next test into one that covers both cases, just so there are fewer snapshots to look at.

Reply:
I guess this one wants both `eager_loading` flavors as well?